README.md

```diff
@@ -3,10 +3,11 @@
 A modified version of [arcreative's homebridge-smartthings-routine](https://github.com/arcreative/homebridge-smartthings-routine) to work
 even after recent API Changes. This is a little plugin to trigger SmartThings
 routines from [Homebridge](https://github.com/nfarina/homebridge).
+The original script has been modified to automatically turn the Homekit switch back off after the delay period set in config.
 
 ## Installation
 
-1. `npm install -g ajvwhite/homebridge-smartthings-routine-triggers`
+1. `npm install -g ajvwhite/homebridge-smartthings-routines`
```

Comment:
Now this is posted in NPM, I really wouldn't change the name at this point - just so people can continue to get the benefit of continued development without changing their references

Reply:
I think this is where the confusion is. Which of these should be used? Also, I'm pretty sure the second one, line 10, isn't actually real because it's under the wrong username.

Reply:
@thayneq the fork hasn't updated the readme correctly - it will be correct when merged in (after edits)